api/suggestion.go

```diff
@@ -0,0 +1,45 @@
+package api
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+
+	"github.com/bitmark-inc/autonomy-api/schema"
+)
+
+// listResources is an API handler for resources listing
+func (s *Server) listResources(c *gin.Context) {
+	account, ok := c.MustGet("account").(*schema.Account)
+	if !ok {
+		abortWithEncoding(c, http.StatusInternalServerError, errorInternalServer)
+		return
+	}
+
+	var params struct {
+		Suggestion bool   `form:"suggestion"`
+		Language   string `form:"lang"`
+	}
+
+	if err := c.Bind(&params); err != nil {
+		abortWithEncoding(c, http.StatusBadRequest, errorInvalidParameters, err)
+		return
+	}
+
+	lang := "en"
+	if params.Language != "" {
+		lang = params.Language
+	}
+
+	if !params.Suggestion {
+		abortWithEncoding(c, http.StatusInternalServerError, errorResourceNotSupport)
```

Comment:
What's `params.Suggestion` for?

Reply:
It is to make this API work like this: `/api/resources?lang=zh&suggestion=true`. We do not have rest resource for `resource` now.